tests/conftest.py

```diff
@@ -300,6 +300,20 @@ def batch_id(self):
                '08f5308af03fd4aa18ff1d868f043b12dd7b0a792e141f000a2505acd4b7a956'
 
 
+class UnfrozenToOperationalTransaction:
+    """
+    Impose unfrozen to opetational balance transaction's data transfer object.
+    """
+
+    @property
+    def batch_id(self):
+        """
+        Get batch identifier of the transaction.
+        """
+        return '37809770b004dcbc7dae116fd9f17428255ddddee3304c9b3d14609d2792e78f' \
+               '08f5308af03fd4aa18ff1d868f043b12dd7b0a792e141f000a2505acd4b7a956'
+
+
```

Comment:
Use the fixture `transaction` instead. All specific transactions will be removed in respect `transaction`.

Reply:
Done.